python/ray/data/_internal/execution/operators/map_operator.py

```diff
@@ -198,7 +199,7 @@ def create(
                 transform_fn, or None to use the block size. Setting the batch size is
                 important for the performance of GPU-accelerated transform functions.
                 The actual rows passed may be less if the dataset is small.
-            supports_fusion: Whether this operator supports fusion with other operators.
+            operator_options: Options for configuring the operator.
```

Comment:
I think this could be confusing to the user, because there are a bunch of options. what are ur thoughts on this 
```suggestion
            operator_options: Advanced options for configuring the operator.
```